nixos/modules/services/x11/desktop-managers/gnome3.nix

```diff
@@ -30,6 +31,9 @@ let
        [org.gnome.desktop.screensaver]
        picture-uri='${pkgs.nixos-artwork.wallpapers.simple-dark-gray-bottom}/share/artwork/gnome/nix-wallpaper-simple-dark-gray_bottom.png'
 
+       [org.gnome.shell]
+       favorite-apps=[ 'org.gnome.Epiphany.desktop', 'evolution.desktop', 'org.gnome.Music.desktop', 'org.gnome.Photos.desktop', 'org.gnome.Nautilus.desktop', 'org.gnome.Software.desktop' ]
```

Comment:
What happens if this gets redefined with `extraGSettingsOverrides` in the same file?

Reply:
I think the later value wins. At least that is what `org.gnome.desktop.background.picture-uri` does for me.

Reply:
Thanks. Just checking because I plan to do something like this with the gnome3 iso.